.github/workflows/build-docker.yaml

```
@@ -0,0 +1,85 @@
+---
+name: Build Docker
+
+on:
+  workflow_call:
+    inputs:
+      build_type:
+        description: Is it a "development" or a "stable" release?
+        required: true
+        type: string
+        default: development
```

actionlint (1.7.4)

input "build_type" of workflow_call event has the default value "development", but it is also required. if an input is marked as required, its default value will never be used

(events)